Interesting; this docker-py test looks to be flaky;

```

[2020-01-03T11:58:55.181Z] =================================== FAILURES ===================================
[2020-01-03T11:58:55.181Z] ____________________ ImageCollectionTest.test_pull_multiple ____________________
[2020-01-03T11:58:55.181Z] tests/integration/models_images_test.py:90: in test_pull_multiple
[2020-01-03T11:58:55.181Z]     assert len(images) == 1
[2020-01-03T11:58:55.181Z] E   AssertionError: assert 2 == 1
[2020-01-03T11:58:55.181Z] E    +  where 2 = len([<Image: 'hello-world:linux'>, <Image: 'hello-world:latest'>])
[2020-01-03T11:58:55.181Z] ------- generated xml file: /src/bundles/test-docker-py/junit-report.xml -------
```

--- a/api/swagger.yaml
+++ b/api/swagger.yaml
@@ -4942,9 +4942,6 @@ paths:
                 type: "string"
               LogPath:
                 type: "string"
-              Node:
-                description: "TODO"
-                type: "object"
               Name:
                 type: "string"
               RestartCount:

--- a/api/types/types.go
+++ b/api/types/types.go
@@ -154,7 +154,7 @@ type Info struct {
 	Images             int
 	Driver             string
 	DriverStatus       [][2]string
-	SystemStatus       [][2]string
+	SystemStatus       [][2]string // SystemStatus is only propagated by the Swarm standalone API
 	Plugins            PluginsInfo
 	MemoryLimit        bool
 	SwapLimit          bool
@@ -318,7 +318,7 @@ type ContainerState struct {
 }
 
 // ContainerNode stores information about the node that a container
-// is running on.  It's only available in Docker Swarm
+// is running on.  It's only used by the Docker Swarm standalone API
 type ContainerNode struct {
 	ID        string
 	IPAddress string `json:"IP"`
@@ -342,7 +342,7 @@ type ContainerJSONBase struct {
 	HostnamePath    string
 	HostsPath       string
 	LogPath         string
-	Node            *ContainerNode `json:",omitempty"`
+	Node            *ContainerNode `json:",omitempty"` // Node is only propagated by Docker Swarm standalone API
 	Name            string
 	RestartCount    int
 	Driver          string

--- a/client/container_inspect_test.go
+++ b/client/container_inspect_test.go
@@ -88,6 +88,8 @@ func TestContainerInspect(t *testing.T) {
 	}
 }
 
+// TestContainerInspectNode tests that the "Node" field is included in the "inspect"
+// output. This information is only present when connected to a Swarm standalone API.
 func TestContainerInspectNode(t *testing.T) {
 	client := &Client{
 		client: newMockClient(func(req *http.Request) (*http.Response, error) {

